test/e2e/kcp_adoption.go

```diff
@@ -52,7 +52,7 @@ type KCPAdoptionSpecInput struct {
 type ClusterProxy interface {
 	framework.ClusterProxy
 
-	ApplyWithArgs(context.Context, []byte, ...string) error
+	Apply(context.Context, []byte, ...string) error
```

Comment:
This change seems to be unrelated, was it intended?

Reply:
a previous PR changed this, so was getting a panic, check Fabrizio comment above

Reply:
Ah I see it now, didn't see the the connection the first time.